crates/core/migrations/2022-05-09-133434_index_name_on_store_config_jsons/up.sql

```diff
@@ -0,0 +1,2 @@
+create index if not exists name_store_config_jsons_idx on
+  store_config_jsons using hash (name);
```

Comment:
I think we want the default index type of btree since we are doing sorts. Hash is good for joining tables on something like an address.

Reply:
Ok